src/current/_includes/releases/v25.2/cluster-setting-changes.md

```diff
@@ -1,49 +1,16 @@
-Changes to [cluster settings]({% link v25.1/cluster-settings.md %}) should be reviewed prior to upgrading. New default cluster setting values will be used unless you have manually set a value for a setting. This can be confirmed by running the SQL statement `SELECT * FROM system.settings` to view the non-default settings.
+Changes to [cluster settings]({% link v25.2/cluster-settings.md %}) should be reviewed prior to upgrading. New default cluster setting values will be used unless you have manually set a value for a setting. This can be confirmed by running the SQL statement `SELECT * FROM system.settings` to view the non-default settings.
 
-- Bullet
-- Bullet
-- Bullet
-- Bullet
-- Bullet
+<h5 id="v25-2-0-settings-added">Settings added</h5>
 
-<h5 id="v25-1-0-settings-added">Settings added</h5>
+- `feature.vector_index.enabled` - Set to `TRUE` to enable vector indexes. Default is `FALSE` (not enabled).
+- `server.child_metrics.include_aggregate.enabled` - When `TRUE`, reports both aggregate and child Prometheus metrics, which can be helpful for quick top-level insights or backward compatibility, but should be disabled if you’re seeing inflated values in Prometheus queries due to double counting. Defaults to `TRUE`.
+- `ui.default_timezone` - Allows you to set the time zone for displayed timestamps in the DB Console. (Refer to [DB Console timezone configuration]({% link v25.2/ui-overview.md %}#db-console-timezone-configuration).) Replaces the deprecated [`ui.display_timezone` cluster setting]({% link v25.2/cluster-settings.md %}#setting-ui-display-timezone). If that value had been set, it will automatically be applied to the new setting `ui.default_timezone`, which takes precedence.
+- `server.oidc_authentication.provider.custom_ca`- Supports a custom root CA for verifying certificates while authenticating with an OIDC provider.
```

Comment:
I'd prefer colons rather than hyphens here, but for consistency:

```suggestion
- `server.oidc_authentication.provider.custom_ca` - Supports a custom root CA for verifying certificates while authenticating with an OIDC provider.
```